### PR TITLE
ETH+/ETH deployment/verification scripts

### PR DESCRIPTION
**Pull request by tbrent**
The main decision is what to use for `defaultThreshold`, which is what will be used to compare ETH+ LP to WETH after accounting for melting appreciation.

### CODE REVIEW

**Review by akshatmittal — APPROVED** [review]

**Comment by tbrent** [comment]
waiting on a second review to merge

**Review by julianmrodri — APPROVED** [review]
LGTM

**Pull request merged**
